### DISCUSSION TITLE
Welcome to usb_cdc Discussions!

### DISCUSSION THREAD

**Discussion by ulixxe** [ulixxe/usb_cdc#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dalnefre** [comment]
This project appears to be exactly what we're looking for. We've been developing a novel processor architecture called [uFork](https://ufork.org/). We've built an [FPGA implementation](https://github.com/organix/uFork/tree/main/fpga) using the Lattice iCE40 UP5K in the Fomu. The current design includes a simple serial debug port on the external pins. We would like to integrate a USB-serial bridge to provide easier access to the processor.

**Reply by ulixxe** [reply]
Interesting! I used usb_cdc to communicate with my [uf16](https://github.com/ulixxe/uf16) 16-bit stack machine.

**Reply by dalnefre** [reply]
Yes. I stumbled across uf16 when browsing your repos and was pleased to see that we both have an appreciation for Forth.

If we run into any difficulties integrating your usb_cdc, in order to keep the "Welcome!" topic clean I assume it would be appropriate to open a new Discussion topic.
